cypress/integration/Images/LeadImageTest.feature

```diff
@@ -0,0 +1,49 @@
+Feature: Lead Image tests
+
+    Scenario Outline: As a user, I want to view the Lead image and Caption on pages with lead image
+        Given user is navigating to "<url>"
+        Then lead image is visible
+        And "<caption>" caption is correct
```

Comment:
This step should be 
And caption text is "\<caption\>"  - now it reads more like a good plain English rather than code piece

Reply:
fixed this